desktop/apps/auction_support/client/bid_form.coffee

```diff
@@ -12,10 +12,12 @@ module.exports = class BidForm extends ErrorHandlingForm
 
   timesPolledForBidPlacement: 0
   maxTimesPolledForBidPlacement: sd.MAX_POLLS_FOR_MAX_BIDS
-  errors:
+  errors: -> {
     "Sale Closed to Bids": "Sorry, your bid wasn't received before the auction closed."
+    "Live Bidding has Started": => "Live bidding on this sale has begun. <br>To continue bidding, please <a href=\"#{@model.liveAuctionUrl()}\">join our Live Auction Room</a>."
```

Comment:
Ah, I was wondering about that as i was copying. Amended.
I guess unclosed `<br>` tags are technically valid html5 but I don't know why. It looks wrong.
http://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br